app/controllers/posts_controller.rb

```diff
@@ -6,7 +6,26 @@ def index
   end
 
   def show
-    @user = User.find(params[:user_id])
-    @post = @user.posts.find(params[:id])
+    @post = Post.find(params[:id])
+    @comments = @post.five_most_recent_comments
+  end
+
+  def new
+    @post = @current_user.posts.new
```

Comment:
- [ ] In this method you're supposed to initialize the Post instead of assigning the @post the current user posts.

Reply:
Hi @mikethreels Thank you for reviewing my code. I have modified my code